3737/0/about_java_interop.rb

```diff
@@ -13,21 +14,20 @@
 # * Calling custom java class
 # * Calling Ruby from java???
 
-#:nodoc:
 class AboutJavaInterop < Neo::Koan
   def test_using_a_java_library_class
     java_array = java.util.ArrayList.new
-    assert_equal Java::JavaUtil::ArrayList, java_array.class
+    assert_equal __, java_array.class
   end
 
   def test_java_class_can_be_referenced_using_both_ruby_and_java_like_syntax
-    assert_equal true, Java::JavaUtil::ArrayList == java.util.ArrayList
+    assert_equal __, Java::JavaUtil::ArrayList == java.util.ArrayList
```

Comment:
read about `asset_equal`
if you want to check if objects are equal you should pass them as params
```
 => a == b
true
```

`assert_equal a, b`